in_winevtlog: Add descriptions about xml query for filtering events (#1179)
fluent/fluent-bit-docs: pipeline/inputs/windows-event-log-winevtlog.md

Note that some Windows Event Log channels \(like `Security`\) requires an admin privilege for reading. In this case, you need to run fluent-bit as an administrator.

#### Query Languages for Event_Query Parameter

> `Event_Query` parameter can handle XML query for filtering Windows EventLog when collecting them.

Reviewer: ```suggestion
`Event_Query` parameter can be used to generate XML query for filtering Windows EventLog during collection.
```

Reviewer: What is an XML query though? That's probably the main thing - XML is a generic marshalling tech so is this something specific?

Author: I have no idea to describe simply about XML query - which can be composable and declarative format of query language for filtering Windows EventLog.

XML Query language is described here: https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events
But, this is the introductory and not fundamental information of the query language.

Reviewer: Ah ok, so XML query is not some special MS thing - it just means query via XML using XPath.
```suggestion
The `Event_Query` parameter can be used to specify the XML query for filtering Windows EventLog during collection.
```

> This parameter is able to handle both of [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) and XML Query queries.

Reviewer: ```suggestion
The supported query types are [XPath](https://developer.mozilla.org/en-US/docs/Web/XPath) and XML Query.
```

Link to XML Query?

Author: XML Query is having some of the specific syntax for querying Windows EventLog. However, I wasn't able to find the suitable page which describes the abstract of the syntax....

> For further description, please refer [the MSDN doc](https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events).

Reviewer: ```suggestion
For further details, please refer to [the MSDN doc](https://learn.microsoft.com/en-us/windows/win32/wes/consuming-events).
```
